src/main/scala/org/allenai/pipeline/s3/S3Artifact.scala

```diff
@@ -150,19 +190,12 @@ object DefaultS3Cache extends LocalS3Cache()
   *                           on the size of the data stored in the persistent directory
   */
 case class LocalS3Cache(
```

Comment:
The changes in this class are basically just putting it back the way it was before I touched it.